ScoutSuite/frontend/src/pages/Findings/index.js

```diff
@@ -27,7 +54,8 @@ const Findings = () => {
         <Breadcrumb />
         <div className="findings">
           <div className="table-card no-items">
-            <CheckCircleOutlineOutlinedIcon /> <b>All good!</b> No findings for this service.
+            <CheckCircleOutlineOutlinedIcon /> <b>All good!</b> No findings for
```

Comment:
didnt Xavier want us to remove the "All good" part?

Reply:
It's in my ohter PR: https://github.com/nccgroup/ScoutSuite/pull/1286